src/js/view/controls/controls.js

```diff
@@ -370,7 +375,7 @@ export default class Controls {
 
         if (this.div.parentNode) {
             removeClass(this.playerContainer, 'jw-flag-touch');
-            this.playerContainer.removeChild(this.div);
+            this.div.parentNode.removeChild(this.div);
```

Comment:
We can save the wrapper in a constant so that we don't have to do `.parentNode` each time